### PR TITLE
Service worker loader comment and preview mode detection cleanup

### PR DESCRIPTION
**Pull request by Bielousov**
**JIRA**: N/A

## Changes
- Refactoring: preview mode detection code in service worker loader should return boolean
- Updated service worker loader intro comment

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that service worker messages are shown as before in the console and that it is shown as registered in the Application tab of Chrome dev tools.

### CODE REVIEW

File: web/service-worker-loader.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-// THIS IS A STRING VALUE NOT A BOOLEAN, BE CAREFUL
-const isPreview = /preview=(true|false)/.exec(self.location.search)[1]
+// MOBIFY PROGRESSIVE SERVICE WORKER LOADER
+// DO NOT MODIFY
```
Comment (jeremywiebe) [inline web/service-worker-loader.js]: When I see comments like this, I usually have two questions:
1. Why can't I modify it?
1. What if I feel I need to modify it?

Comment (Bielousov) [inline web/service-worker-loader.js]: Because this file is sent to client and the comment is for them.

Comment (jeremywiebe) [inline web/service-worker-loader.js]: Oh, I see now. I'm not 100% sure. 🤔 

Maybe just leave it the way it is.

Comment (DanielBrookRoberge) [inline web/service-worker-loader.js]: Maybe say `DO NOT MODIFY WITHOUT APPROVAL FROM MOBIFY`?

**Pull request merged**